# Encapsulates filtered, initiated, or double gestures
std_msgs/Header header
string event_type             # "filtered", "initiated", "double"
string gesture_name
bool is_left         